lib/OcrProcessors/OcrMyPdfBasedProcessor.php

```diff
@@ -33,6 +33,11 @@
 use Psr\Log\LoggerInterface;
 
 abstract class OcrMyPdfBasedProcessor implements IOcrProcessor {
+	private static $ocrModes = [
```

Comment:
At this point you can consider to change the name, since it is not only about modes, but about a mapping to arguments.